include/glow/Importer/ONNXIFILoader.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef GLOW_IMPORTER_ONNXIFILOADER_H
+#define GLOW_IMPORTER_ONNXIFILOADER_H
+
+#include "onnx/onnxifi.h"
+
+#include "glow/Importer/ONNX.h"
+
+namespace glow {
+namespace onnxifi {
+
+class ModelLoader : public ONNXModelLoader {
```

Comment:
I'm not sure about the naming here -- I would initially assume from the names that a ModelLoader would not be ONNX specific, and that ONNXModelLoader would inherit from ModelLoader, adding in ONNX-specific functionality.

Reply:
it's in glow::onnxifi namespace hence onnxifi specific.